pkg/controller/volume/persistentvolume/controller.go

```diff
@@ -964,7 +964,7 @@ func (ctrl *PersistentVolumeController) recycleVolumeOperation(arg interface{})
 	volume = newVolume
 
 	// Find a plugin.
-	spec := vol.NewSpecFromPersistentVolume(volume, false)
+	spec := vol.NewSpecFromPersistentVolume(volume, false, "")
```

Comment:
in this method specifically, we really don't know anything about the namespace. For instance, a volume could be RWM (ie, nfs) and thus associated with multiple namespaces. I think it does not make sense to have namespace-specific actions in recycle.